packages/components/src/local/atoms/game-title/index.stories.tsx

```diff
@@ -1,13 +1,26 @@
 import React from 'react'
+import { styled } from '@material-ui/core/styles'
+import Box from '@material-ui/core/Box'
 
 // Import component files
 import GameTitle from './index'
 import docs from './README.md'
 
```

Comment:
@zukilover - I _think_ we use `BlueContainer` in a number of stories.  Should we define a single instance somewhere centrally?   That would make it easier to update if the higher level styling changes.

Reply:
Blue container moved to global preview configuration